LICENSE

```diff
@@ -289,3 +289,12 @@ Copyright: 2011-2018 The Apache Software Foundation
 Home page: https://spark.apache.org/
 License: https://www.apache.org/licenses/LICENSE-2.0
 
+--------------------------------------------------------------------------------
+
+This product includes code from Delta Lake.
+
+* AssignmentAlignmentSupport is an independent development but was inspired by an equivalent rule in Delta.
```

Comment:
Can you be more specific about "inspired by"? Did you use the code from Delta as a reference? Did you use the same test cases?

Reply:
It was used as a reference. I did not simply copy the code as is so I added "inspired by" to give credit to the original authors. The rules should be logically equivalent and should be part of Spark, not data sources.

Reply:
Updated.